src/main/resources/sentry.properties

```diff
@@ -1 +1,37 @@
-dsn=
+# TODO: https://pihemr.atlassian.net/browse/OBPIH-6417 We can't currently set DSN via openboxes config files such
+#       as application.yml. Once we figure out how to allow that, delete this file from the openboxes repo then add
+#       sentry.properties to the .gitignore. We should be treating this file as if it's an override (like a .env file).
+
+# According to Sentry documentation, this isn't a total Sentry feature switch. Some behaviour may still leak
+# through. To fully disable Sentry, you need to unset the Sentry DSN.
+enabled=true
+
+# If you ever need to debug Sentry changes locally, you can set the following properties. Note that any values
+# defined here will be overridden by "SENTRY_DSN" and "SENTRY_ENVIRONMENT" environment variables if those are defined.
+dsn=${SENTRY_DSN_BACKEND}
```

Comment:
See https://github.com/openboxes/openboxes-devops/pull/74 for details on how this env var is being set.